app/config.py

```diff
@@ -395,6 +398,7 @@ class Staging(Config):
     NOTIFY_ENVIRONMENT = 'staging'
     CSV_UPLOAD_BUCKET_NAME = 'staging-notify-csv-upload'
     LETTERS_PDF_BUCKET_NAME = 'staging-letters-pdf'
+    TEST_LETTERS_BUCKET_NAME = 'staging-test-letters'
```

Comment:
We should probably prefix all of these with the environment at some point

Reply:
Not sure what you mean here, they are already prefixed with the environment?